src/components/rooms-tab.js

```diff
@@ -0,0 +1,153 @@
+import by from 'thenby';
+import React from 'react';
+import { Table } from 'antd';
+import { useUser } from './user-context.js';
+import { useTranslation } from 'react-i18next';
+import { useDateFormat } from './language-context.js';
+
+const fakeGetRoomUrl = roomId => `/rooms/${roomId}`;
+const getOwnedRooms = user => [
+  {
+    key: 'dsadasdsa',
+    name: 'Room 1',
+    createdOn: new Date().toISOString(),
+    access: 'public',
+    owner: user
+  }, {
+    name: 'Room 2',
+    key: 'dsfdsdsf',
+    createdOn: new Date().toISOString(),
+    access: 'private',
+    owner: user
+  }
+];
+
+const getMembershipRooms = user => [
+  {
+    key: 'dsadasdsa',
+    name: 'Room 1',
+    createdOn: new Date().toISOString(),
+    access: 'public',
+    owner: {
+      username: 'user1',
+      email: 'user1@test.com'
+    },
+    members: [{ userId: user._id, joinedOn: new Date().toISOString() }]
+  }, {
+    name: 'Room 2',
+    key: 'dsfdsdsf',
+    createdOn: new Date().toISOString(),
+    access: 'private',
+    owner: {
+      username: 'user2'
+    },
+    members: [{ userId: user._id, joinedOn: new Date().toISOString() }]
+  }
+];
+
+function RoomsTab() {
+  const user = useUser();
+  const { formatDate } = useDateFormat();
+  const { t } = useTranslation('roomsTab');
+
+  const renderName = (_value, room) => {
+    return <a href={fakeGetRoomUrl(room._id)}>{room.name}</a>;
+  };
+
+  const renderCreatedOn = (_value, room) => {
+    return <span>{formatDate(room.createdOn)}</span>;
+  };
+
+  const renderOwner = (_value, room) => {
+    return room.owner.email
+      ? <span>{room.owner.username} | <a href={`mailto:${room.owner.email}`}>{t('email')}</a></span>
+      : <span>{room.owner.username}</span>;
+  };
+
+  const renderJoinedOn = (_value, room) => {
+    const userAsMember = room.members.find(member => member.userId === user._id);
+    return <span>{formatDate(userAsMember.joinedOn)}</span>;
+  };
+
+  const renderAccess = (_value, room) => {
+    return <span>{t(`accessType_${room.access}`)}</span>;
```

Comment:
The _value is already the access level as defined by the data key in the column below

Reply:
True! I perpetuated the inconsistencies and confusions to this page 🤦‍♀️ 
Corrected, thanks!

Reply:
By the way, I streamlined and corrected the other tables too.

Reply:
:+1: